src/test/java/pl/zimi/zadanka/Zad5Test.java

```diff
@@ -0,0 +1,72 @@
+package pl.zimi.zadanka;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Zad5Test {
+    @Test
+    void n0() {
+        Zad5 object = new Zad5();
+        Assertions.assertEquals(0, object.climbStairs(0));
+    }
+
+    @Test
+    void n3() {
+        Zad5 object = new Zad5();
+        Assertions.assertEquals(3, object.climbStairs(3));
+    }
+
+    @Test
+    void n3TreeVersion() {
+        Zad5 object = new Zad5();
+        Assertions.assertEquals(3, object.climbStairsTreeStyle(3, 0, 0));
```

Comment:
metoda w testach powinna mieć tylko jeden parametr...

dwa kolejne parametry to jakieś internalowe rzeczy...
je chcemy ukrywać (enkapsulować) nie chcemy żeby ktoś tu mógł wrzucić -1

tą metodę możemy ukryć za pomocą private... żebyśmy tylko wewnątrz klasy mogli jej użyć

Reply:
ok utworzyłem kolejną metodę typu public która wywołuje metodę właściwą, której typ zmieniłem na private.